functions.php

```diff
@@ -43,3 +43,20 @@ function get_template_content($name = null){
 }
 
 add_action('wp_enqueue_scripts', 'register_custom_script');
+
+// Allow Gutenberg core blocks
+function p4_child_theme_gpap_whitelist_blocks( $allowed_blocks, $post ) {
+	$allowed = is_array($allowed_blocks) ? $allowed_blocks : array();
+  array_push($allowed, 'core/cover');
+	array_push($allowed, 'core/post-title');
+  array_push($allowed, 'core/post-excerpt');
+  array_push($allowed, 'core/post-featured-image');
+  array_push($allowed, 'core/post-content');
+  array_push($allowed, 'core/post-author');
+  array_push($allowed, 'core/post-date');
+  array_push($allowed, 'core/post-modified-date');
+  array_push($allowed, 'core/post-categories');
+  array_push($allowed, 'core/post-tags');
+	return $allowed;
+}
+add_filter('allowed_block_types', 'p4_child_theme_gpap_whitelist_blocks', 11, 2);
```

Comment:
better name 

```suggestion
add_filter('allowed_block_types', 'p4_child_theme_gpap_add_allowed_blocks', 11, 2);
```

see https://www.vice.com/en/article/we-need-to-stop-saying-blacklist-and-whitelist/